AppPackage/Sources/PromiseManagement/ConfirmedDetail/ConfirmedDetailView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ConfirmedDetailView.swift
+//
+//
+//  Created by Sujin Jin on 2023/03/06.
+//
+
+import ComposableArchitecture
+import DesignSystem
+import SwiftUI
+
+public struct ConfirmedDetailFeature: ReducerProtocol {
+    public struct State: Equatable {
+        public let id: UUID
+        let title: String
+        let theme: String
+        let date: String
+        let place: String
+        let participants: [String]
+
+        init(_ cellState: ConfirmedCell.State) {
+            id = cellState.id
+            title = cellState.title
+            theme = cellState.theme
+            date = cellState.date
+            place = cellState.place
+            participants = cellState.participants
+        }
+    }
+
+    public enum Action: Equatable {
+        case dismissed
+    }
+
+    public var body: some ReducerProtocol<State, Action> {
+        Reduce { _, action in
+            switch action {
+            case .dismissed:
+                return .none
+            }
+        }
+    }
+}
+
+// MARK: - ConfirmedDetailView
+
+public struct ConfirmedDetailView: View {
+    private let illustImage = "illustDetail"
+    private let closeImage = "iconClose"
```

Comment:
Image는 View이기 때문에 아래 처럼 변경해도 좋을것 같아요 (마이너한 의견)
```suggestion
    private let illustImageName = "illustDetail"
    private let closeImageName = "iconClose"
```

Reply:
확실히 뷰라고 인식할 수도 있겠다는 생각이 드네요!  반영하겠습니다!